store/src/java/com/zimbra/cs/db/DbVolume.java

```diff
@@ -329,15 +336,15 @@ public static boolean isVolumeReferenced(DbConnection conn, short volumeId) thro
 
     private static boolean isVolumeReferenced(DbConnection conn, short volumeId, int groupId) throws ServiceException {
         return tableRefsVolume(conn, volumeId, groupId, "revision") || tableRefsVolume(conn, volumeId, groupId, "revision_dumpster") ||
-            tableRefsVolume(conn, volumeId, groupId, "mail_item_dumpster") || tableRefsVolume(conn, volumeId, groupId, "mail_item");
+                tableRefsVolume(conn, volumeId, groupId, "mail_item_dumpster") || tableRefsVolume(conn, volumeId, groupId, "mail_item");
```

Comment:
Indentation issue

Reply:
I think this is the right, split statement should always be right to the above keyword.